src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -0,0 +1,258 @@
+            text={<Trans>Disconnect</Trans>}
+          />
+        </IconContainer>
+      </div>
+      <Column>
+        <div style={{ padding: '16px 0' }}>
+          <Text fontSize={36} fontWeight={400}>
+            {balanceString} {nativeCurrencySymbol}
+          </Text>
+          <USDText>${amountUSD.toFixed(2)} USD</USDText>
+        </div>
+        {unclaimedAmount !== undefined && unclaimedAmount?.toFixed(0, { groupSeparator: ',' } ?? '-') !== '0' && (
+          <UNIbutton style={{ border: 'none' }}>
+            <Trans>Claim</Trans> {unclaimedAmount?.toFixed(0, { groupSeparator: ',' } ?? '-')} <Trans>reward</Trans>
+          </UNIbutton>
+        )}
+      </Column>
+    </>
+  )
+}
+
+const PendingBadge = styled.span`
+  background-color: #4c82fb3d;
+  color: #869eff;
+  font-weight: 600;
+  padding: 4px 6px;
+  border-radius: 5px;
+`
+
+const IconWrap = styled.span`
+  margin-top: auto;
+  margin-bottom: auto;
+  margin-left: 4px;
+  height: 16px;
+  width: 16px;
+`
+
+const DefaultText = styled(Text)`
+  font-size: 14px;
+  font-weight: 400;
+`
+
+const WalletDropdown = ({ setMenu }: { setMenu: (state: 'DEFAULT' | 'LANGUAGE' | 'TRANSACTIONS') => void }) => {
+  const { account } = useWeb3React()
+  const isAuthenticated = !!account
+  const [darkMode, toggleDarkMode] = useDarkModeManager()
+  const activeLocale = useActiveLocale()
+  const ISO = activeLocale.split('-')[0].toUpperCase()
+  const allTransactions = useAllTransactions()
+  const toggleWalletModal = useToggleWalletModal()
+
+  const pendingTransactions = useMemo(
+    () => Object.values(allTransactions).filter((tx) => !tx.receipt),
+    [allTransactions]
+  )
+
+  return (
+    <div style={{ paddingLeft: 16, paddingRight: 16, width: '100%', height: '100%' }}>
+      {isAuthenticated ? (
+        <AuthenticatedHeader />
+      ) : (
+        <ConnectButton onClick={toggleWalletModal}>Connect wallet</ConnectButton>
+      )}
+      <Divider />
+      {isAuthenticated && (
+        <ToggleMenuItem onClick={() => setMenu('TRANSACTIONS')}>
+          <DefaultText>
+            <Trans>Transactions</Trans>{' '}
+            {pendingTransactions.length > 0 && (
+              <PendingBadge>
+                {pendingTransactions.length} <Trans>Pending</Trans>
+              </PendingBadge>
+            )}
+          </DefaultText>
+          <IconWrap>
+            <ChevronRight size={16} strokeWidth={3} />
+          </IconWrap>
+        </ToggleMenuItem>
+      )}
+      <ToggleMenuItem onClick={() => setMenu('LANGUAGE')}>
+        <DefaultText>
+          <Trans>Language</Trans>
+        </DefaultText>
+        <span style={{ display: 'flex' }}>
+          <span style={{ marginTop: 'auto', marginBottom: 'auto' }}>
+            <DefaultText>{ISO}</DefaultText>
+          </span>
+          <IconWrap>
+            <ChevronRight size={16} strokeWidth={3} />
+          </IconWrap>
+        </span>
+      </ToggleMenuItem>
+      <ToggleMenuItem onClick={toggleDarkMode}>
```

Comment:
you'll want to add one more for feature flags

Reply:
<img width="433" alt="Screen Shot 2022-08-17 at 2 15 51 PM" src="https://user-images.githubusercontent.com/15934595/185212947-9d611af0-94dd-44a3-bd4b-b6d857725c77.png">


Feature flags are being moved here, so I don't think we need them in wallet